Nziranziza/Send-IT
# #162020760 add UI templates to the project repository

**What does this PR do?**

Add UI templates to the project repository

**Description of task to be completed?**

- Copy files from previous ui template into project repo
- Deploy it to github pages

**How should this be tested manually**

Once you visit the project repository you will see the UI directory in develop branch
In branches you will gh-pages and this shows that UI templates are hosted to github pages

**Any background context you want to provide**

No,there isn't

**What are the relevant pivotal tracker stories?**

#162020760
